src/renderer/components/monaco-editor/monaco-editor.tsx

```diff
@@ -55,22 +55,18 @@ export class MonacoEditor extends React.Component<MonacoEditorProps> {
   public staticId = `editor-id#${Math.round(1e7 * Math.random())}`;
   public dispose = disposer();
 
-  // TODO: investigate how to replace with "common/logger"
-  //  currently leads for stucking UI forever & infinite loop.
-  //  e.g. happens on tab change/create, maybe some other cases too.
-  logger = console;
-
   @observable.ref containerElem: HTMLElement;
   @observable.ref editor: editor.IStandaloneCodeEditor;
```

Comment:
Shouldn't these also be private?

Reply:
Maybe `protected`, but not sure.. @jakolehm any suggestions?

Reply:
I think these ones should be public as if developer will decide to make a `ref={monaco => monaco.editor}` those would be available.